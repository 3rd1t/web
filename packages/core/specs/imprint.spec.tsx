import React from "react"
import { render } from "@testing-library/react"

import Imprint from "../pages/imprint"

jest.mock("next/router", () => ({
  useRouter() {
    return {
      push: jest.fn(),
    }
  },
}))
describe("Imprint", () => {
  it("should render successfully", () => {
    const { baseElement } = render(<Imprint />)
    expect(baseElement).toBeTruthy()
  })
})
