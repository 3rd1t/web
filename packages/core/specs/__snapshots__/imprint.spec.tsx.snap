// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Imprint should render successfully 1`] = `
<body>
  <div>
    <div>
      <div
        class="pt-16 bg-gray-100"
      >
        <div
          class="fixed inset-x-0 top-0 z-20 bg-gray-100"
        >
          <nav
            class="container flex flex-wrap items-center justify-between h-16 px-4 mx-auto md:flex-no-wrap"
            role="navigation"
          >
            <a
              href="/"
            >
              <div
                class="flex items-center space-x-2"
              >
                <svg
                  class="h-6 text-gray-900 stroke-current"
                  fill="none"
                  viewBox="0 0 194 148"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M9 106.208L97.3883 17.8199L134.158 54.5894M185.07 41.8615L96.6814 130.25L59.9118 93.4803"
                    stroke-width="25"
                  />
                </svg>
                <div
                  class="text-xl font-semibold"
                >
                  <span
                    class="text-gray-800"
                  >
                    PERFOLIO
                  </span>
                </div>
              </div>
            </a>
            <div
              class="hidden md:block"
            >
              <div
                class="w-auto"
              >
                <ul
                  class="flex items-center space-x-10"
                >
                  <li>
                    <a
                      class="text-lg font-medium text-gray-700 transition duration-150 ease-in-out hover:text-black"
                      href="/"
                    >
                      Product
                    </a>
                  </li>
                  <li>
                    <a
                      class="text-lg font-medium text-gray-700 transition duration-150 ease-in-out hover:text-black"
                      href="/#features"
                    >
                      Features
                    </a>
                  </li>
                  <li>
                    <a
                      class="text-lg font-medium text-gray-700 transition duration-150 ease-in-out hover:text-black"
                      href="/#team"
                    >
                      Team
                    </a>
                  </li>
                  <li>
                    <a
                      class="text-lg font-medium text-gray-700 transition duration-150 ease-in-out hover:text-black"
                      href="/docs"
                    >
                      Documentation
                    </a>
                  </li>
                </ul>
              </div>
            </div>
            <div
              class="block md:hidden"
            >
              <div>
                <button
                  aria-label="open menu"
                  class="p-2 text-gray-700 hover:text-gray-900 focus:outline-none"
                  data-testid="toggle"
                >
                  <svg
                    class="w-6 h-6"
                    fill="none"
                    stroke="currentColor"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M4 6h16M4 12h16M4 18h16"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="2"
                    />
                  </svg>
                </button>
              </div>
            </div>
          </nav>
        </div>
        <section>
          <div
            class="min-h-screen"
          >
            <div
              class="container px-4 py-10 mx-auto md:py-12 lg:py-16 xl:py-20 xl:px-0"
            >
              <section>
                <div
                  class="mx-auto prose-sm prose sm:prose lg:prose-lg xl:prose-xl"
                >
                  <div
                    class="container px-4 py-10 mx-auto md:py-12 lg:py-16 xl:py-20 xl:px-0"
                  >
                    <span
                      class="text-sm "
                    >
                      German only
                    </span>
                    <h1>
                      Imprint / Impressum
                    </h1>
                    <h3>
                      Angaben gemäß § 5 TMG
                    </h3>
                    <span>
                      Webersinke Thomas Jordt Kohler GbR
                    </span>
                    <br />
                    <span>
                      Treibberg 9
                    </span>
                    <br />
                    <span>
                      90403 Nürnberg
                    </span>
                    <h3>
                      Vertreten durch:
                    </h3>
                    <div
                      class="flex flex-col"
                    >
                      <span>
                        Nicolas Webersinke
                      </span>
                      <span>
                        Andreas Thomas
                      </span>
                      <span>
                        Mads Jordt
                      </span>
                      <span>
                        Kevin Kohler
                      </span>
                    </div>
                    <h3>
                      Kontakt
                    </h3>
                    <div
                      class="flex flex-col"
                    >
                      <span>
                        Telefon: +49 (0) 160 155 1312
                      </span>
                      <span>
                        E-Mail: info@perfol.io
                      </span>
                    </div>
                    <h3>
                       EU-Streitschlichtung
                    </h3>
                    <p>
                      Die Europäische Kommission stellt eine Plattform zur Online-Streitbeilegung (OS) bereit: https://ec.europa.eu/consumers/odr. Unsere E-Mail-Adresse finden Sie oben im Impressum.
                    </p>
                    <h3>
                      Verbraucherstreitbeilegung / Universalschlichtungsstelle
                    </h3>
                    <p>
                      Wir sind nicht bereit oder verpflichtet, an Streitbeilegungsverfahren vor einer Verbraucherschlichtungsstelle teilzunehmen.
                    </p>
                    <h3>
                      Haftung für Inhalte
                    </h3>
                    <p>
                      Als Diensteanbieter sind wir gemäß § 7 Abs.1 TMG für eigene Inhalte auf diesen Seiten nach den allgemeinen Gesetzen verantwortlich. Nach §§ 8 bis 10 TMG sind wir als Diensteanbieter jedoch nicht verpflichtet, übermittelte oder gespeicherte fremde Informationen zu überwachen oder nach Umständen zu forschen, die auf eine rechtswidrige Tätigkeit hinweisen.
                    </p>
                    <p>
                      Verpflichtungen zur Entfernung oder Sperrung der Nutzung von Informationen nach den allgemeinen Gesetzen bleiben hiervon unberührt. Eine diesbezügliche Haftung ist jedoch erst ab dem Zeitpunkt der Kenntnis einer konkreten Rechtsverletzung möglich. Bei Bekanntwerden von entsprechenden Rechtsverletzungen werden wir diese Inhalte umgehend entfernen.
                    </p>
                    <h3>
                      Haftung für Links
                    </h3>
                    <p>
                      Unser Angebot enthält Links zu externen Websites Dritter, auf deren Inhalte wir keinen Einfluss haben. Deshalb können wir für diese fremden Inhalte auch keine Gewähr übernehmen. Für die Inhalte der verlinkten Seiten ist stets der jeweilige Anbieter oder Betreiber der Seiten verantwortlich. Die verlinkten Seiten wurden zum Zeitpunkt der Verlinkung auf mögliche Rechtsverstöße überprüft. Rechtswidrige Inhalte waren zum Zeitpunkt der Verlinkung nicht erkennbar.
                    </p>
                    <p>
                      Eine permanente inhaltliche Kontrolle der verlinkten Seiten ist jedoch ohne konkrete Anhaltspunkte einer Rechtsverletzung nicht zumutbar. Bei Bekanntwerden von Rechtsverletzungen werden wir derartige Links umgehend entfernen.
                    </p>
                    <h3>
                      Urheberrecht
                    </h3>
                    <p>
                      Die durch die Seitenbetreiber erstellten Inhalte und Werke auf diesen Seiten unterliegen dem deutschen Urheberrecht. Die Vervielfältigung, Bearbeitung, Verbreitung und jede Art der Verwertung außerhalb der Grenzen des Urheberrechtes bedürfen der schriftlichen Zustimmung des jeweiligen Autors bzw. Erstellers. Downloads und Kopien dieser Seite sind nur für den privaten, nicht kommerziellen Gebrauch gestattet.
                    </p>
                    <p>
                      Soweit die Inhalte auf dieser Seite nicht vom Betreiber erstellt wurden, werden die Urheberrechte Dritter beachtet. Insbesondere werden Inhalte Dritter als solche gekennzeichnet. Sollten Sie trotzdem auf eine Urheberrechtsverletzung aufmerksam werden, bitten wir um einen entsprechenden Hinweis. Bei Bekanntwerden von Rechtsverletzungen werden wir derartige Inhalte umgehend entfernen.
                    </p>
                    <p>
                      Quelle: 
                      <a
                        href="https://www.e-recht24.de"
                      >
                        https://www.e-recht24.de
                      </a>
                    </p>
                  </div>
                </div>
              </section>
            </div>
          </div>
        </section>
      </div>
      <footer
        class="bg-carbon-900"
      >
        <div
          class="container mx-auto"
        >
          <div
            class="flex flex-wrap justify-between p-10  text-gray-100"
          >
            <div
              class="p-2 md:space-y-2"
            >
              <span
                class="text-xs font-semibold text-white uppercase"
              >
                Projects
              </span>
              <a
                class="block focus:outline-none text-gray-100 hover:text-white"
                href="https://perfol.io"
              >
                Home
              </a>
              <div
                class="flex items-center space-x-2"
              >
                <a
                  class="block focus:outline-none text-gray-100 hover:text-white"
                  href="https://analytics.perfol.io"
                >
                  Analytics
                </a>
                <span
                  class="px-1 text-sm bg-purple-800 rounded-sm"
                >
                  WIP
                </span>
              </div>
              <a
                class="block focus:outline-none text-gray-100 hover:text-white"
                href="https://research.perfol.io"
              >
                Research
              </a>
            </div>
            <div
              class="p-2 md:space-y-2"
            >
              <span
                class="text-xs font-semibold text-white uppercase"
              >
                Resources
              </span>
              <a
                class="block focus:outline-none text-gray-100 hover:text-white"
                href="/docs"
              >
                Documentation
              </a>
              <a
                class="block focus:outline-none text-gray-100 hover:text-white"
                href="/imprint"
              >
                Imprint
              </a>
              <a
                class="block focus:outline-none text-gray-100 hover:text-white"
                href="/privacy"
              >
                Privacy
              </a>
            </div>
            <div
              class="p-2 md:space-y-2"
            >
              <span
                class="text-xs font-semibold text-white uppercase"
              >
                Contact us
              </span>
              <a
                class="block focus:outline-none text-gray-100 hover:text-white"
                href="mailto:info@perfol.io"
              >
                contact@perfol.io
              </a>
            </div>
            <div
              class="p-2 md:space-y-2"
            >
              <span
                class="text-xs font-semibold text-white uppercase"
              >
                Source
              </span>
              <a
                class="block focus:outline-none text-gray-100 hover:text-white"
                href="https://github.com/perfolio"
              >
                github.com
              </a>
            </div>
          </div>
        </div>
        <div
          class="pt-2 border-t border-carbon-700"
        >
          <div
            class="max-w-6xl mx-auto"
          >
            <div
              class="flex flex-col items-center px-10 py-2 text-sm md:flex-row"
            >
              <span
                class="text-white"
              >
                © Copyright 
                2020
                . All Rights Reserved.
              </span>
              <div
                class="flex flex-row md:flex-auto md:flex-row-reverse"
              >
                <a
                  href="https://www.netlify.com"
                >
                  <img
                    alt="Deploys by Netlify"
                    src="https://www.netlify.com/img/global/badges/netlify-light.svg"
                  />
                </a>
              </div>
            </div>
          </div>
        </div>
      </footer>
    </div>
  </div>
</body>
`;
